packages/storybook/src/react-components/select/select.stories.tsx

```diff
@@ -0,0 +1,102 @@
+import { LuxSelect, LuxSelectOption, type LuxSelectOptionProps } from '@lux-design-system/components-react';
+import tokens from '@lux-design-system/design-tokens/dist/index.json';
+import type { Meta, StoryObj } from '@storybook/react';
+
+type Story = StoryObj<typeof meta>;
+
+const meta = {
+  title: 'React Components/Select',
+  id: 'react-components-select',
+  component: LuxSelect,
+  subcomponents: {},
+  parameters: {
+    tokens,
+    tokensPrefix: 'utrecht-select',
+  },
+  argTypes: {
+    disabled: {
+      description: 'disabled indicator',
+      control: 'boolean',
+    },
+    required: {
+      description: 'required indicator',
+      control: 'boolean',
+    },
+    invalid: {
+      description: 'invalid indicator',
+      control: 'boolean',
+    },
+  },
+} satisfies Meta<typeof LuxSelect>;
+
+export default meta;
+
+const selectOptions: LuxSelectOptionProps[] = [
+  { value: '', label: 'Select an option', disabled: true },
+  { value: '1', label: 'Option #1' },
+  { value: '2', label: 'Option #2' },
+  { value: '3', label: 'Option #3' },
+  { value: '4', label: 'Option #4' },
+];
+
+const SelectTemplate: Story = {
+  args: {},
+  render: ({ ...args }) => (
+    <LuxSelect {...args}>
+      {selectOptions &&
```

Comment:
Deze 2 checks zijn niet nodig, toch? `selectOptions` is hard-coded dus we weten dat deze bestaat en een aantal opties bevat.